config/dr-cluster/manifests/bases/ramen_dr_cluster.clusterserviceversion.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   annotations:
     alm-examples: '[]'
     capabilities: Basic Install
+    operators.openshift.io/infrastructure-features: '["disconnected"]'
```

Comment:
Downstream requires this annotation, which technically can be added from the build scripts, but non-trivial (as it is an edit between the file). It is feasible to do this via a kustomization, but left as is for now.